packages/account-v2/src/modules/AddressFields/validations.ts

```diff
@@ -0,0 +1,66 @@
+import * as Yup from 'yup';
+
+const regexChecks = {
+    address_details: {
+        address_city: /^\p{L}[\p{L}\s'.-]{0,99}$/u,
+        address_line_1: /^[\p{L}\p{Nd}\s'.,:;()\u00b0@#/-]{1,70}$/u,
+        address_line_2: /^[\p{L}\p{Nd}\s'.,:;()\u00b0@#/-]{0,70}$/u,
+        address_postcode: /^[a-zA-Z0-9\s-]{0,20}$/,
+        address_state: /^[\w\s\W'.;,-]{0,99}$/,
+        non_jersey_postcode: /^(?!\s*je.*)[a-zA-Z0-9\s-]*/i,
+    },
+};
+
+const addressPermittedSpecialCharactersMessage = ". , ' : ; ( ) ° @ # / -";
+
+export const addressDetailValidations = (countryCode: string, isSvg: boolean) => ({
```

Comment:
can we have test cases for these validation schema

Reply:
Done.